packages/expo-sms/android/src/main/java/expo/modules/sms/SMSModule.kt

```diff
@@ -0,0 +1,134 @@
+package expo.modules.sms
+
+import android.content.Context
+import org.unimodules.core.ExportedModule
+import org.unimodules.core.interfaces.LifecycleEventListener
+import org.unimodules.core.ModuleRegistry
+import org.unimodules.core.Promise
+import org.unimodules.core.interfaces.services.UIManager
+import org.unimodules.core.interfaces.ExpoMethod
+import java.util.ArrayList
+import android.content.Intent
+import android.net.Uri
+import android.provider.Telephony
+import android.content.pm.PackageManager
+import android.os.Bundle
+import org.unimodules.core.interfaces.ActivityProvider
+import java.lang.StringBuilder
```

Comment:
```suggestion
import android.content.Context
import android.content.Intent
import android.content.pm.PackageManager
import android.net.Uri
import android.provider.Telephony
import android.os.Bundle

import java.util.ArrayList
import java.lang.StringBuilder

import org.unimodules.core.ExportedModule
import org.unimodules.core.ModuleRegistry
import org.unimodules.core.Promise
import org.unimodules.core.interfaces.ActivityProvider
import org.unimodules.core.interfaces.ExpoMethod
import org.unimodules.core.interfaces.LifecycleEventListener
import org.unimodules.core.interfaces.services.UIManager
```

We need some linter for kotlin files 🤔 @lukmccall, @tsapeta we need to consider adding some in the nearest future 😓

Reply:
We actually have one, but in Expo Go - https://github.com/expo/expo/blob/30bfe260516b5c4bb3324f8617e561c0e0f50874/android/build.gradle#L120-L124

I'll add it also to the bare-expo